ocs_ci/utility/pull_images_for_logging_42/pull_images.py

```diff
@@ -0,0 +1,69 @@
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+
+import os
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def pull_packages_from_quay():
+    """
+    This is the function to pull the ocp 4.2 package manifests
+    for cluster-logging resources
+    """
+
+    registry = 'aosqe42'
+    version = "4.2"
+    path = "ocs_ci/utility/pull_images_for_logging_42"
```

Comment:
I think since these are temporary scripts, its better we keep this in private github repo and reference the repo and run it from there. It will keep our repo clean. what do you say?

Reply:
@vasukulkarni We may face the same issue when we switch over to 4.3 and where ocp 4.3 is not be GAed. Since openshift-logging is does not come along with the builds probably we may be using the same scripts to pull 4.3 images with little changes here and there, That is the reason of adding these scripts to our repo.

Reply:
I think we should discuss more on this approach. we have to put the shell scripts and all logic outside our repo mainly because

a) complexity of shell script, documentation and maintenance
b) is independent of our CI logic.
c) can be maintained and merged independently ( I will create a different repo name in red-hat-storage)

Reply:
@Shrivaibavi I have this repo now, we can move the pull-image logic there and use the script from that location. https://github.com/red-hat-storage/quay-pull-images

Reply:
@vasukulkarni Will move the scripts to the repo in next commit

Reply:
Moved the scripts to the repo https://github.com/red-hat-storage/quay-pull-images